src/pages/instances/EditInstance.tsx

```diff
@@ -85,7 +88,9 @@ export type EditInstanceFormValues = InstanceEditDetailsFormValues &
   MigrationFormValues &
   BootFormValues &
   CloudInitFormValues &
-  YamlFormValues;
+  YamlFormValues & {
+    editRestriction?: string;
+  };
```

Comment:
better add this to the `InstanceEditDetailsFormValues` and `InstanceDetailsFormValues`.